components/rating/index.css

```
@@ -74,37 +99,163 @@ governing permissions and limitations under the License.
+      inset-inline-end: calc((var(--mod-rating-icon-width, var(--spectrum-rating-icon-width)) - var(--mod-rating-indicator-width, var(--spectrum-rating-indicator-width))) / 2);
     }
   }
 
   /* All stars following the hovered star */
   &:hover ~ .spectrum-Rating-icon {
-    @extend %spectrum-Rating-strokedStar;
+    .spectrum-Rating-starActive {
+      display: none;
+    }
+    .spectrum-Rating-starInactive {
+      display: block;
+    }
+  }
+
+  /* focus-ring */
+  &:after {
+    content: '';
+    position: absolute;
+    inset: calc(-2 * var(--mod-rating-focus-indicator-gap, var(--spectrum-rating-focus-indicator-gap)));
+
+    opacity: 0;
+
+    border-width: var(--mod-rating-focus-indicator-thickness, var(--spectrum-rating-focus-indicator-thickness));
+    border-style: solid;
+    border-color: var(--mod-rating-focus-indicator-color, var(--spectrum-rating-focus-indicator-color));
+    border-radius: var(--mod-rating-border-radius, var(--spectrum-rating-border-radius));
   }
 }
 
 .spectrum-Rating-starActive,
 .spectrum-Rating-starInactive {
-  inline-size: var(--spectrum-icon-star-width);
-  block-size: var(--spectrum-icon-star-height);
   margin: 0 auto;
 }
+
+.spectrum-Rating {
+  &.is-focused {
+
+    box-shadow: 0 0 0 var(--mod-rating-focus-indicator-thickness, var(--spectrum-rating-focus-indicator-thickness)) var(--mod-rating-focus-indicator-color, var(--spectrum-rating-focus-indicator-color));
+
+  }
+
+  &:hover {
+    .spectrum-Rating-icon {
+      /* Make all stars colored when the component is hovered */
+      color: var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default));
+
+      &:hover {
+        /* The icon that's actually hovered is darker */
+        color: var(--mod-rating-icon-color-hover, var(--spectrum-rating-icon-color-hover));
+      }
+
+      &:active {
+        /* The icon that's actually hovered is darker */
+        color: var(--mod-rating-icon-color-down, var(--spectrum-rating-icon-color-down));
+      }
+
+      /* Show which star is currently selected */
+      &.is-currentValue {
+        &:after {
+          /* Match the color of the star */
+          background: currentColor;
+        }
+      }
+    }
+  }
+}
+
+.spectrum-Rating-icon {
+  color: var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default));
+
+  &.is-selected {
+    color: var(--mod-rating-icon-color-key-focus, var(--spectrum-rating-icon-color-key-focus));
+  }
+
+  &:hover {
+    /* All stars following the hovered star */
+    & ~ .spectrum-Rating-icon {
+      color: var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default));
+    }
+  }
+}
+
+.spectrum-Rating--emphasized {
+  .spectrum-Rating-icon {
+    color: var(--mod-rating-icon-color-default, var(--spectrum-rating-icon-color-default));
+
+    &.is-selected {
+      color: var(--mod-rating-emphasized-icon-color-key-focus, var(--spectrum-rating-emphasized-icon-color-key-focus));
```

I think you may be using the incorrect token here.

```suggestion
      color: var(--mod-rating-emphasized-icon-color-default, var(--spectrum-rating-emphasized-icon-color-default));
```

---

I fixed this.